candidate-standard/abstract_tests/queries/ATS_area.adoc

```diff
@@ -0,0 +1,75 @@
+[[ats_area]]
+[width="90%",cols="2,6a"]
+|===
+^|*Abstract Test {counter:ats-id}* |*/ats/area*
+^|Test Purpose |Validate that an error is returned by a `Area` query if no query parameters are specified.
+^|Requirement |<<req_area,/req/queries/area>>
+^|Test Method |. No query parameters are specified
+. Validate that a document was returned with a status code 404.
+
+^|*Abstract Test {counter:ats-id}* |*/ats/area*
+^|Test Purpose |Validate that an error is returned by a `Area` query when the coords query parameter is not specified.
+^|Requirement |<<req_area,/req/queries/area>>
+^|Test Method |. coords query parameter is not specified
+. Validate that a document was returned with a status code 404.
+
+^|*Abstract Test {counter:ats-id}* |*/ats/area*
+^|Test Purpose |Validate that an error is returned by a `Area` query when the coords query parameter does not contain a valid POLYGON Well Known Text value.
+^|Requirement |<<rreq_position,/req/queries/position>>
+^|Test Method |. Check coords query parameter is a valid Well Known Text Polygon value 
+. Validate that a document was returned with a status code 403.
```

Comment:
403 forbidden? Why not just 400?

Reply:
I will update the document to change it to a 400 error